tools/packaging/guest-image/build_image.sh

```diff
@@ -45,6 +45,7 @@ build_initrd() {
 		AGENT_TARBALL="${AGENT_TARBALL}" \
 		AGENT_INIT="yes" \
 		AGENT_POLICY="${AGENT_POLICY:-}" \
+		PULL_TYPE="${PULL_TYPE:-}" \
```

Comment:
Please, either declare PULL_TYPE=${PULL_TYPE:-default} at the beginng of the file, please, or set it as PULL_TYPE:-default here.

Reply:
Fixed.